var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -69,6 +69,12 @@ def install(self, spec, prefix):
             '--with-blas-libs=%s' % ' '.join(blas.names),
             '--with-blas-lib-dirs=%s' % ' '.join(blas.directories)
         ]
+        
+        if spec.satisfies('^netlib-lapack'):
+            # netlib-lapack puts the libs in lib64
+            configure_args.extend([
+                "--with-lapack-lib-dirs=%s/lib64".join(lapack.directories),
```

Comment:
Can you explain how to do this in netlib-lapack package? It seems I have to use setup_dependent_package right?